server/controllers/authController.js

```diff
@@ -1,93 +1,61 @@
+import dotenv from 'dotenv';
+import { Pool } from 'pg';
```

Comment:
@T2Wil  this should be in the database file

Reply:
solved!